apps/server/lib/lexical/server/code_intelligence/completion/translations/struct.ex

```diff
@@ -20,6 +20,10 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Struct do
     end
   end
 
+  def completion(%Env{} = env, builder, module_name, full_name, 0) do
+    completion(env, builder, module_name, full_name)
+  end
+
   def completion(%Env{} = env, builder, module_name, full_name, more) when is_integer(more) do
```

Comment:
why not just make `more \\ 0` ?

Reply:
we have a function below, it will cause conflicts.

`  def completion(%Env{} = env, builder, struct_name, full_name) do`